CONTRIBUTING.md

```diff
@@ -26,54 +26,23 @@ cabal v2-build Cabal:unit-tests # build Cabal's unit test suite
 cabal v2-build cabal-tests # etc...
 ~~~~
 
-**Dogfooding HEAD.**
```

Comment:
This is still valid though, isn't it?

Reply:
I think it's confusing to keep, since it implies having to think about something special. It is now the _only_ way we test `cabal-install`, so i don't think it makes sense to point out specifically.